### PR TITLE
Expand the docs on package adding/updating

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,7 +92,12 @@ Using a local git checkout, change `source_dir` as appropriate:
 1. Choose a template from gem2rpm that's suitable for the type of package and
    run:
   `./add_gem_package.sh GEM_NAME TEMPLATE TITO_TAG`
-1. Add the package to `package_manifest.yaml`. Amend the commit.
+1. Add the package to `package_manifest.yaml`
+1. Add a changelog entry
+  1. Find `%changelog`
```
Comment (ekohl) [inline README.md]: This renders other than I expected. Updating

File: README.md [hunk 2]
```diff
@@ -119,25 +126,50 @@ In both cases:
...
 1. Commit the changes
   1. `git commit -m "Update NAME to VERSION"`
 1. Follow the "test a package" section above until it builds for all
    targeted platforms and required SCL + non-SCL modes.
 1. Submit a pull request against `rpm/develop`
 
+### Updating npm packages
+
+1. Run the update script
+  1. `./add_npm_pacakge.sh PACKAGE`
```
Comment (xprazak2) [inline README.md]: s/pacakge/package/

**Comment by ekohl** [comment]
Updated with correct nested lists and fixed the typo.

**Comment by ekohl** [comment]
Corrected the changelog part

**Comment by ekohl** [comment]
Would it make more sense to include a helper script? I know there's the idea of using ansible for this, but in the short term a helper script could be very effective.

**Comment by evgeni** [comment]
@ekohl I would not be opposed to a `sed` or `ansible -m lineinfile` oneliner to do that. (But I, personally, also would not need one ;))

**Comment by ekohl** [comment]
@evgeni I updated some scripts to include this automatically and the docs should now reflect this.

File: README.md [hunk 3]
```diff
@@ -114,26 +117,38 @@ In both cases:
...
-## HOWTO: update a gem package
+## HOWTO: Updating packages
+
+### Updating gem packages
+
+1. Run the update script
+   * `./bump_rpm.sh rubygem-foo`
+1. Check the output
+   * update `Requires` to match changes in runtime dependencies
+   * add/remove entries in %files if required for new root files
```
Comment (evgeni) [inline README.md]: `%files` should be in backticks?

Comment (ekohl) [reply README.md]: Fixed

**Pull request merged**